packages/flame/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:flutter/gestures.dart';
+
+import '../../extensions.dart';
+import '../game/game.dart';
+
+class EventPosition {
+  final Game _game;
+  final Offset _localPosition;
+  final Offset? _globalPosition;
+
+  late final Vector2 game = _game.projectCoordinates(_localPosition);
+  late final Vector2 widget = _localPosition.toVector2();
+  late final Vector2 global = _globalPosition?.toVector2() ??
+      _game.convertLocalToGlobalCoordinate(_localPosition.toVector2());
+
+  EventPosition(this._game, this._localPosition, this._globalPosition);
+}
+
+abstract class _BaseInfo<T> {
+  final T raw;
+
+  _BaseInfo(this.raw);
+}
+
+abstract class _PositionInfo<T> extends _BaseInfo<T> {
+  final Game _game;
+  final Offset _position;
+  final Offset? _globalPosition;
+
+  late final position = EventPosition(
+    _game,
+    _position,
+    _globalPosition,
+  );
```

Comment:
Couldn't the getters of `EventPosition` live directly in here?
Maybe a personal preference but I found it a bit confusing with `info.position.game/widget/global`, might not be clearer directly on here though, just a thought.

Reply:
They could, but to be honest, I liked it to live on its own "namespace", specially because it makes sense when we add position + game, or local, etc.

But no strong opinions too, I wouldn't mind adding the getters here if we decide so.

Reply:
Or maybe if `position` was named something else which makes it clearer that it contains many "positions"?
What do you think?

Reply:
sounds good! But I can't think on any better name than `position` atm, any suggestions?

Reply:
I prefer them to be namespaced as well, that "forces" users to think about it and choose the right one. also, if any event has multiple positions, it's all neatly wraped